docling/cli/main.py

```diff
@@ -279,12 +284,18 @@ def convert(
     if from_formats is None:
         from_formats = [e for e in InputFormat]
 
+    parsed_headers: Optional[Dict[str, str]] = None
+    if headers is not None:
+        parsed_headers = json.loads(headers)
```

Comment:
we could use pydantic to parse the dict which will guarantee we don't get weird/unsupported structures.

```suggestion
        headers_t = TypeAdapter(Dict[str, str])
        parsed_headers = headers_t.validate_json(headers)
```